pip/rules.bzl

```diff
@@ -181,7 +173,7 @@ def _deploy_pip_impl(ctx):
             "{package_file}": ctx.attr.target[PyDeploymentInfo].package.short_path,
             "{wheel_file}": ctx.attr.target[PyDeploymentInfo].wheel.short_path,
             "{version_file}": ctx.attr.target[PyDeploymentInfo].version_file.short_path,
-            "{pypi_profile}": ctx.attr.pypi_profile,
```

Comment:
`pypi` is not a term in the python packaging standards, this actually intends to refer to a `pyprc` repository name

Reply:
Rename to `pypirc`